draft-ietf-rats-eat.md

```diff
@@ -367,23 +367,42 @@ aggregation and type definition feature of CDDL). In the data model,
 described in the {{datamodel}}, the CDDL groups turn into CBOR map
 entries and JSON name/value pairs.
 
-## Nonce Claim (cti and jti)
 
-All EATs should have a nonce to prevent replay attacks. The nonce is
-generated by the relying party, provided to the entity,
-and included in the token. Note that intrinsically by the nature of a
-nonce no security is needed for its transport.
+## Token ID Claim (cti and jti)
 
-CWT defines the "cti" claim. JWT defines the "jti" claim. These carry
-the nonce in an EAT.
+CWT defines the "cti" claim. JWT defines the "jti" claim. These are
+equivalent to each other in EAT and carry a unique token identifier as
+they do in JWT and CWT.  They may be used to defend against re use of
+the token but are distinct from the nonce that is used by the relying
+party to guarantee freshness and defend against replay.
 
-TODO: what about the JWT claim "nonce"?
 
 ## Timestamp claim (iat)
 
 The "iat" claim defined in CWT and JWT is used to indicate the
 date-of-creation of the token.
 
+
+## Nonce Claim (nonce)
+
+All EATs should have a nonce to prevent replay attacks. The nonce is
+generated by the relying party, the end consumer of the token. It is
+conveyed to the entity over whatever transport is in use before the
+token is generated and then included in the token as the nonce claim.
+Note that intrinsically by the nature of a nonce no security is needed
```

Comment:
It is not clear what this means to me.  Besides, in the light of the text proposed in #28
```
[...] Since an EAT may be created by an entity that may not support the same type of transport security as the consumer of the EAT, intermediaries may be required to bridge communications between the entity and consumer.  As a result, it is RECOMMENDED that both the consumer create a nonce, and the entity leverage the nonce along with COSE mechanisms for encryption and signing to create the EAT
```
this should probably be dropped?

Reply:
Hi Thomas,

Have read this over a few times, but I don't know what is not clear to you and what you think should be dropped.

Reply:
(Apologies for the late response -- for some reasons I didn't get the nudge from
GitHub on your reply).   The thing that I don't understand is `Note that
intrinsically by the nature of a nonce no security is needed`.  In fact, I think
it's quite the opposite: the nonce is core to the security of the attestation?
My suggestion is to simply drop the `Note [...] no security is needed`.